node-fs-writefile/write-rand-promise.js

```diff
@@ -0,0 +1,8 @@
+import { writeFile } from 'node:fs/promises';
+import { Buffer } from 'node:buffer';
+
+const data = new Uint8Array(Buffer.from('Hello LFZ'));
+writeFile('./random.txt', data)
+  .then((value) => Math.random());
```

Comment:
What's the point of this `.then`? It doesn't look like it does anything.

I think you want to assign a random number to `data`.